src/Base/Ragnarok/CharServer.pm

```diff
@@ -233,7 +233,7 @@ sub sendCharInfo {
 
 			my $sex;
 
-			if($masterServer->{charBlockSize} == 145 && $masterServer->{serverType} =~ /^iRO/) {
+			if($masterServer->{charBlockSize} >= 145) {
```

Comment:
well ?

Reply:
all char block size >= 145 have sex in struct, we dont need to filter by server